src/lib/pubkey/frodokem/frodo_constants.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * FrodoKEM constants
+ * Based on the MIT licensed reference implementation by the designers
+ * (https://github.com/microsoft/PQCrypto-LWEKE/tree/master/src)
```

Comment:
Is anything in this file actually derived from the reference implementation?

Reply:
Fixed